src/main/kotlin/me/serce/solidity/lang/types/inference.kt

```diff
@@ -166,22 +164,30 @@ fun inferExprType(expr: SolExpression?): SolType {
       inferExprType(expr.expressionList[1])
     )
     is SolFunctionCallExpression -> {
-      expr.expressionList.firstOrNull().let { first ->
-        if (expr.expressionList.size == 1 && first is SolPrimaryExpression && first.elementaryTypeName != null) {
-          getSolType(first.elementaryTypeName)
+      val cast = if (expr.functionCallArguments.expressionList.size == 1) {
```

Comment:
🤔 I wonder why `expr.functionCallArguments` isn't classified as nullable even though it's annotated as `@Nullable`:
```java
public interface SolFunctionCallExpression extends SolExpression, SolFunctionCallElement {

  @NotNull
  SolExpression getExpression();

  @Nullable
  SolFunctionCallArguments getFunctionCallArguments();
}
```